packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -188,22 +188,28 @@ export default class PerformanceMonitoring {
         task.eventType === 'click'
       ) {
         const target = task.target
-        const name = target.getAttribute('name')
+        const tagName = target.tagName.toLowerCase()
 
-        let additionalInfo = ''
-        if (name) {
-          additionalInfo = `["${name}"]`
+        // use custom html attribute 'data-transaction-name' - otherwise fall back to "tagname" + "name"-Attribute
+        let transactionName = ''
+        if (!!target.dataset.transactionName) {
+          transactionName = target.dataset.transactionName
+        } else {
+          const name = target.getAttribute('name')
+          if (!!name) {
+            transactionName = `${tagName}["${name}"]`
+          } else {
```

Comment:
one small suggestion, we can set the transactionName to tagName during initialization. 

```
let transactionName = tagName

```